### DISCUSSION TITLE
How can i know which processes uses which ports ?

### DISCUSSION THREAD

**Discussion by ghost** [GyulyVGC/sniffnet#247] [General]
I am using Sniffnet and what i see are the packets being sent from and received by my computer.

What if i wanna know which apps and processes are sending and receiving those packets ?

I have some apps sending and receiving stuff (even when firefox is closed) and i wanna know what are those apps ?

Operating System: Kubuntu 23.04
KDE Plasma Version: 5.27.4
KDE Frameworks Version: 5.104.0
Qt Version: 5.15.8
Kernel Version: 6.2.0-20-generic (64-bit)
Graphics Platform: X11
Processors: 4 × Intel® Core™ i5-7200U CPU @ 2.50GHz
Memory: 23.1 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 620
Manufacturer: Dell Inc.
Product Name: Inspiron 15-3567

Is this feature of showing the link between the ports and the apps using them available in Sniffnet ?

Do you plan to add such feature if it does not exist ?

**Comment by GyulyVGC** [comment]
Hi @curt-wise 

First of all, thanks for your interest in Sniffnet.

As stated in #170, process recognition is still not implemented in the app.

If all your applications are closed and you are still receiving packets, they are probably coming from your local network (e.g. between your network card and your router) or they may be due to some automatic system update.

**Reply by ghost** [reply]
Thanks a lot

**Reply by GyulyVGC** [reply]
No problem!
I remain available for any other doubts or clarifications.
